src/vm/ceemain.cpp

```diff
@@ -930,7 +930,7 @@ void EEStartupHelper(COINITIEE fFlags)
         PerfLog::PerfLogInitialize();
 #endif //ENABLE_PERF_LOG
 
-#ifdef FEATURE_PERFMAP
+#if defined(FEATURE_PERFMAP) && !defined(CROSSGEN_COMPILE)
```

Comment:
It may be better to not define FEATURE_PERFMAP when compiling crossgen (we do it for number of other FEATURE_XXX).